CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.0.1 (2019-07-24)
+
+- Updated package dependency structure to list `rxjs` as a peer dependency.
```

Comment:
I would start with "Fixed..." to justify the patch version.

Reply:
We should also have "the" before "package." ... And should there be a link to a related PR?

Reply:
Good point :) thanks!